### ISSUE TITLE
Updating the nightly version of rust gives error on module path:

### ISSUE THREAD

**Issue by amiyatulu** [rustwasm/wasm-bindgen#4115]
While importing the module it now gives following error:

```rust
#[wasm_bindgen(module = "/src/package.js")]
extern "C" {
    #[wasm_bindgen]
    pub fn helloworld();
```
```
error: constant evaluation is taking a long time
   --> /home/amiya/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/slice/mod.rs:118:9
    |
118 |         ptr::metadata(self)
    |         ^^^^^^^^^^^^^^^^^^^
    |
    = note: this lint makes sure the compiler doesn't get stuck due to infinite loops in const eval.
            If your compilation actually takes a long time, you can safely allow the lint.
help: the constant being evaluated
   --> src/js_extension_binding.rs:5:1
    |
5   | #[wasm_bindgen(module = "/src/package.js")]
    | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    = note: `#[deny(long_running_const_eval)]` on by default
    = note: this error originates in the attribute macro `wasm_bindgen` (in Nightly builds, run with -Z macro-backtrace for more info)

note: erroneous constant encountered
 --> src/js_extension_binding.rs:5:1
  |
5 | #[wasm_bindgen(module = "/src/package.js")]
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this note originates in the attribute macro `wasm_bindgen` (in Nightly builds, run with -Z macro-backtrace for more info)

```

**Comment by lucasmerlin** [comment]
I'm having the same problem, but for me it came after updating wasm_bindgen from 0.2.92 to 0.2.93. So I guess it's a combination of wasm_bindgen 0.2.93 and a recent rust nightly version (I'm using `1.82.0-nightly (2c93fabd9 2024-08-15)`)? 
@amiyatulu do you know which nightly version you were updating from?

**Comment by daxpedda** [comment]
I'm unable to reproduce it, maybe it was addressed in the latest nightly already?
Let me know if this is still an issue and I can reopen!

**Comment by lucasmerlin** [comment]
Still happening with the current nightly :/ 

```rust
error: constant evaluation is taking a long time
   --> /Users/lucasprivat/.rustup/toolchains/nightly-aarch64-apple-darwin/lib/rustlib/src/rust/library/core/src/slice/mod.rs:118:9
    |
118 |         ptr::metadata(self)
    |         ^^^^^^^^^^^^^^^^^^^
    |
    = note: this lint makes sure the compiler doesn't get stuck due to infinite loops in const eval.
            If your compilation actually takes a long time, you can safely allow the lint.
help: the constant being evaluated
   --> hp-mobile-interop/src/web.rs:23:3
    |
23  |   #[wasm_bindgen(module = "/js/out/index.js")]
    |   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    = note: `#[deny(long_running_const_eval)]` on by default
    = note: this error originates in the attribute macro `wasm_bindgen` (in Nightly builds, run with -Z macro-backtrace for more info)

note: erroneous constant encountered
  --> hp-mobile-interop/src/web.rs:23:3
   |
23 |   #[wasm_bindgen(module = "/js/out/index.js")]
   |   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   |
   = note: this note originates in the attribute macro `wasm_bindgen` (in Nightly builds, run with -Z macro-backtrace for more info)

```

The js module is quite big (almost 1mb) maybe it's related to that?

**Comment by daxpedda** [comment]
Thank you for getting back on this, I was able to reproduce it with this.
It should be fixed with #4146.

**Comment by lucasmerlin** [comment]
Awesome, this fixed it for me!
